clang/test/CodeGenHLSL/convergence/entry.point.hlsl

```diff
@@ -0,0 +1,11 @@
+// RUN: %clang_cc1 -triple spirv-pc-vulkan-compute -finclude-default-header -fnative-half-type -disable-llvm-passes -emit-llvm -o - %s | FileCheck %s
```

Comment:
This test doesn't do anything with `native half types why do you need the `-fnative-half-type` flag?

Reply:
I copied from another test. I have removed it.